### ISSUE TITLE
Failed to clone after following instructions and creating deploy key

### ISSUE THREAD

**Issue by blm** [GitJournal/GitJournal#640]
Hi,
I just paid for and downloaded the app on ios 15.5 iphone 12.  I followed the app's instructions to create a repository, generate a key and add it as a deploy key with write access.  When I clicked on the "Clone repo" button I got:
```

Failed
MissingPluginException(No implementation found for method gitClone on channel gitjournal.io/git)
```

Thanks!

**Comment by vHanda** [comment]
Sorry. I messed up an update. I think adding a note before configuring the remote might help.

The bug fix is stuck in review.

**Comment by neo1908** [comment]
I have the same issue. I have tried on multiple git hosts using repos that have files and ones that don't

IPhone 13 Pro Max
IOS 15.5

**Comment by steverhall** [comment]
Yes, creating a note first, then adding the GitHub repo worked for me.

**Comment by bmcquist** [comment]
I was seeing this as well. After adding a note, and then adding a GitHub repo, I don't see the Failed message... but the app is stuck on the cloning step for multiple minutes and never finishes. On iOS 15.5, SE gen 2.

**Comment by neo1908** [comment]
Ahh, I got it working. Creating a note **in the app** does work around the issue.

**Comment by MalcolmMielle** [comment]
Similar to @bmcquist im stuck during the cloning at 87.69%. The app has been unusable for me on iOS for more than a month now without a free version :/.

**Comment by vHanda** [comment]
@bmcquist : could you please paste your debug log over here? Settings -> debug logs. Or use the report a bug button?

**Comment by vHanda** [comment]
Closing this issue as it's fixed with 1.85.2.

@bmcquist : could you please open a new issue about the lagging?

**Comment by MalcolmMielle** [comment]
@vHanda thanks for the quick answer. I have opened #642 to follow up on the problem :)
